### DISCUSSION TITLE
Undefined method 'can' in lumen

### DISCUSSION THREAD

**Discussion by silvesterwali** [spatie/laravel-permission#1748] [Q&A] [answered]
lumen version:7.2^
hi there. I'm new to using lumen from laravel . I have followed the documentation from https://spatie.be/docs/laravel-permission/v4/introduction . i the beginning all work just fine, like give and revoke permission, and also assign a role to the user.
but when I try to user permission directly from the controller like below 
![image](https://user-images.githubusercontent.com/40487188/119454131-c25d2780-bd6a-11eb-8462-a8b8179da6dd.png)

here is the User modal class

![image](https://user-images.githubusercontent.com/40487188/119454577-431c2380-bd6b-11eb-8412-6fe5f6aebc76.png)

my vscode show the error.  thank's for your replay  and explanation

**Comment by drbyte** [comment, marked as answer]
Your issue is a Lumen issue.
See https://lumen.laravel.com/docs/8.x/authorization

**Reply by silvesterwali** [reply]
thank's @drbyte
